doc/general/load_checkpoints.md

```
@@ -0,0 +1,13 @@
+# How to load checkpoints
+
+**Summary:** This page provides instructions on how to load specific scenarios in the Carla simulator to avoid having to wait for a specific event to occur.
+
+
+## Instruction
```

Markdownlint (0.37.0)

Expected: 1; Actual: 0; Below
Headings should be surrounded by blank lines

(MD022, blanks-around-headings)